### ISSUE TITLE
[Accessibility] The Calendar/ "Date range" Filter is inaccessible through the keyboard alone

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4964]
## Describe the bug
- The Date range filter in the "Map View" is not keyboard-focusable and operable.Users are unable to choose a range for the calendar filter using the keyboard alone.

## To Reproduce
1. Launch the local host
2. Navigate to the "Map View"
3. Find the date range filters under the Map View and try to interact with it without using the mouse.
4. Observe if you are able to complete these actions using the keyboard only.

## Expected behavior
Date range filter should be keyboard-focusable and operable, allowing users to interact with it

## Screenshots

![Screenshot from 2024-06-18 18-10-19](https://github.com/ushahidi/platform-client-mzima/assets/124133577/c825d7ef-5db9-4aa0-aa17-c4d25bff9b30)

**Comment by Angamanga** [comment]
Closing but keeping in Linear https://linear.app/ushahidi/issue/USH-1438/[accessibility]-the-calendar-date-range-filter-is-inaccessible-through
